features/cli/oc_set_deployment_hook.feature

```diff
@@ -13,80 +13,83 @@ Feature: set deployment-hook/build-hook with CLI
   @heterogeneous @arm64 @amd64
   Scenario: OCP-11805:Workloads Set pre/mid/post deployment hooks on deployment config via oc set deployment-hook
     Given I have a project
-    When I run the :new_app client command with:
-      | template | rails-postgresql-example |
+    When I run the :create_deploymentconfig client command with:
+      | image | quay.io/openshifttest/hello-openshift:multiarch |
```

Comment:
@zhouying7780 if we are changing this to have hello-openshift image then we need to make sure "oc new-app rails-postgres-example' command works fine because i remember there was a bug from a customer about this not working since we show this as an example when creating new project in oc. thanks !!

Reply:
@kasturinarra  firstly this is quickstart feature  , it's not belongs to workloads team's feature , second  I checked the related case and automation script both were set with inactive , so I think we replace it should be okay.

Reply:
okay, thanks !!